sencha-workspace/SlateAdmin/app/controller/people/Invite.js

```diff
@@ -181,5 +192,32 @@ Ext.define('SlateAdmin.controller.people.Invite', {
                 }
             }
         });
+    },
+
+    getDefaultClass: function(callback) {
+        var me = this,
+            store = me.getPeopleInvitationsStore(),
+            classesStore = me.getPeopleUserClassesStore(),
+            cls = store.getModel().getField('UserClass').getDefaultValue();
+
+        if (!cls) {
+            return classesStore.load({
+                callback: function(classes, request, success) {
+                    var response = Ext.decode(request.getResponse().responseText),
+                        defaultCls = response.default || null;
+
+                    if (success) {
+                        store.getModel().getField('UserClass').defaultValue = defaultCls;
+                        me.getInvitationsWindow().on('afterrender', function() {
+                            me.getUserClassCombo().getStore().loadRecords(classes);
```

Comment:
add the store to the controller's stores list and have the combo reference it by string. That way the controller has a getter for accessing the store directly that's outside the lifecycle of the UI (i.e. you can load data into it at any time and not worry about when any thing using it gets rendered)